controllers/common.go

```diff
@@ -51,18 +51,17 @@ func setPodTemplateSpecDefaults(template *corev1.PodTemplateSpec) {
 	if template.Spec.SchedulerName == "" {
 		template.Spec.SchedulerName = "default-scheduler"
 	}
-	// for each volumes, if volumeSource is Projected or SecretVolumeSource, set default mode
+	// for each volumes, if volumeSource is Projected or SecretVolumeSource, set default permission
 	for i := range template.Spec.Volumes {
 		if template.Spec.Volumes[i].Projected != nil {
 			if template.Spec.Volumes[i].Projected != nil {
-				template.Spec.Volumes[i].Projected.DefaultMode = common.DefaultModePtr()
+				template.Spec.Volumes[i].Projected.DefaultMode = ptr.To(common.DefaultPermission)
```

Comment:
from what I searched, this is used if using CloudStorage

Reply:
Yes we need to test this using the CloudStorage API for bucket creation and then use it in DPA BSL configuration. https://docs.openshift.com/container-platform/4.15/backup_and_restore/application_backup_and_restore/aws-sts/oadp-aws-sts.html

Reply:
removed change in Projected volume